cmd/cli.go

```diff
@@ -36,18 +35,23 @@ func Command() {
 				Value: 10,
 				Usage: "Time for polling resource deletion status in seconds",
```

Comment:
I just noticed that the shorthand for `polltime` is `p`, but it's also the shorthand for `project` 😟

Reply:
Haha! I didn't spot that. I'll fix